draft-ietf-rtcweb-jsep.xml

```diff
@@ -2565,6 +2594,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                       connection, if one exists.[TODO: may need revision, i.e., use of actpass</t>
                     <t>DTLS fingerprint values, where at least one fingerprint
                       MUST be present.</t>
+		    <t>All RID reference in an "a=simulcast" line must
```

Comment:
This section is called "Semantics Verification" and has the verification of several other values.  It seems consistent.  Can you be more specific about where you would like it moved?